docs: pod overrides, better install instructions, improved wording (#569)
stackabletech/hbase-operator: docs/modules/hbase/pages/usage-guide/operations/pod-placement.adoc

        replicas: 2
----

> WARNING: The Pods get stuck in `Pending`, if your Kubernetes cluster does not have any nodes without a masters already running on it and sufficient compute resources available.

Reviewer: ```suggestion
WARNING: The Pods remain in the  `Pending` phase until the masters are up and running and there are  sufficient compute resources available.
```
